cleanlab_studio/utils/tlm_lite.py

```diff
@@ -110,12 +116,20 @@ def prompt(
         """
         prompt_response = self._tlm_response.prompt(prompt)
 
-        if isinstance(prompt, Sequence) and isinstance(prompt_response, list):
-            response = [r["response"] for r in prompt_response]
+        if (
+            isinstance(prompt, Sequence)
+            and isinstance(prompt_response, list)
+            and all(r["response"] is not None for r in prompt_response)
+        ):
+            response = cast(List[str], [r["response"] for r in prompt_response])
             perplexity = [r["log"]["perplexity"] for r in prompt_response]
             return self._batch_score(prompt, response, perplexity)
 
-        elif isinstance(prompt, str) and isinstance(prompt_response, dict):
+        elif (
+            isinstance(prompt, str)
+            and isinstance(prompt_response, dict)
+            and prompt_response["response"] is not None
```

Comment:
Same question as above re `prompt_response["response"]` being `None`

Reply:
added this to pass mypy checks